docs/concepts/rest_apis.rst

```diff
@@ -0,0 +1,32 @@
+edx-platform REST API Concepts
```

Comment:
This seems to just be about authentication, and not the many other REST API topics, including those in https://openedx.atlassian.net/wiki/spaces/AC/pages/18350757/Open+edX+REST+API+Conventions. Maybe adjust the title?

Reply:
Cross linked that page here.  I think this will get flesh out more but Auth is kind of the first hurdle so starting there.